Dockerfile.release

```diff
@@ -0,0 +1,30 @@
+FROM phusion/baseimage
+
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US.UTF-8
+ENV LC_ALL en_US.UTF-8
+
+# required packages
+RUN apt-get update && apt-get upgrade -y && apt-get install -y \
+    bash \
+    bash-completion \
+    curl \
+    dnsutils \
+    vim && \
+    apt-get clean
+
+ENV FENNEC_CLIENT_BIND_ADDR=0.0.0.0 FENNEC_CLIENT_PORT=8443 MIX_ENV=prod \
+    REPLACE_OS_VARS=true SHELL=/bin/bash
+
+WORKDIR /opt/app
+
+ADD fennec.tar.gz ./
+
+# Move priv dir
+RUN mv $(find lib -name fennec-*)/priv .
+RUN ln -s $(pwd)/priv $(find lib -name fennec-*)/priv
```

Comment:
Nope. The whole point is to enable mounting this dir as docker volumen. You can't do that with symlinks or I guess links in general but not sure about that.